app/mailers/provider_mailer.rb

```diff
@@ -90,6 +90,24 @@ def fallback_sign_in_email(provider_user, token)
     )
   end
 
+  def application_submitted_over_5_days(provider_user, application_choice)
+    @application = map_application_choice_params(application_choice)
+    email_for_provider(
+      provider_user,
+      application_choice.application_form,
+      subject: I18n.t!('provider_mailer.application_submitted_over_5_days.subject', candidate_name: @application.candidate_name),
+    )
+  end
+
+  def application_submitted_with_safeguarding(provider_user, application_choice)
+    @application = map_application_choice_params(application_choice)
+    email_for_provider(
+      provider_user,
+      application_choice.application_form,
+      subject: I18n.t!('provider_mailer.application_submitted_with_safeguarding.subject', candidate_name: @application.candidate_name),
```

Comment:
No need to pass `candidate_name` here or above if it's not in the i18n string

Reply:
Those subject are just something I made up (and though they may change) but I checked with Laura and we won't need to include candidate's name so removed them.